assembly/util.ts

```diff
@@ -1,20 +1,17 @@
 export function last<T>(arr: T[]): T {
-  return arr[arr.length - 1];
+  return unchecked(arr[arr.length - 1]);
 }
 
 export function first<T>(arr: T[]): T {
-  return arr[0];
+  return unchecked(arr[0]);
 }
 
 export function toArray<T>(item: T): T[] {
-  const arr = new Array<T>();
-  arr.push(item);
-  return arr;
+  return [item];
```

Comment:
For some reason I didn't think Array literals were supported in AS! that does render this helper pretty much redundant